### DISCUSSION TITLE
Links in DM's not clickable

### DISCUSSION THREAD

**Discussion by gosbank** [pixelfed/pixelfed#5131] [Support]
I have version  v0.11.8 and cannot click links (or tap on mobile) in DM's. There are link icons next to the links which display as only the host (eg. www.facebook.com)  The only other issue I've noticed is that avatars are usually blank for remote users, not sure if related issue though. The account DM'ing me in this case is another local user on my instance.

Screenshot: https://pixygram.com/i/web/post/591210862449811316

Server: Ubuntu 20.04
PHP: 8.1
RAM: 4gb
Web Server: Apache

**Comment by gosbank** [comment]
Here is the meta information from "direct_mention" table in database:

{"domain":"www.facebook.com","local":false}

**Comment by ghost** [comment]
Still the case on `0.11.9-3cb50af8`.
